library/src/mixins/property/property_file_split/abc_fy.py

```diff
@@ -1,18 +1,28 @@
 """fy
+property property_file_split: PropertyFileSplit
+"""
+
+import abc
 from typing import List
 
+from pydantic import BaseModel
 
-property property_file_split: List[str]
-"""
+from domain.python_entity_name import PythonEntityName
 
-from typing import List
-import abc
+
+class PropertyFileSplitModel(BaseModel):
+    user_imports: str
+    python_class_name: PythonEntityName
+    property_name: PythonEntityName
+    implementation_name: PythonEntityName
```

Comment:
Let's move creation of `PythonEntityName` to library/src/mixins/property/parsed_property_fy_py_file/using_parsed_fy_py_file_fy.py